K/Kerberos_MIT_krb5/build_tarballs.jl

```diff
@@ -0,0 +1,35 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "MIT_kerberos_krb5"
+version = v"1.19.3"
+
+# Collection of sources required to complete build
+sources = [
+ArchiveSource("https://kerberos.org/dist/krb5/1.19/krb5-1.19.3.tar.gz", "56d04863cfddc9d9eb7af17556e043e3537d41c6e545610778676cf551b9dcd0")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/krb5-1.19.3/src
```

Comment:
Sorry, I missed this before: we usually avoid hardcoding version numbers in these path, otherwise we need to change them every time we want to update a package
```suggestion
cd $WORKSPACE/srcdir/krb5*/src
```